recipes/readerwriterqueue/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, CMake, tools
+from fnmatch import fnmatch
+import os
+
+
+class ReaderWriterQueue(ConanFile):
+    name = "readerwriterqueue"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/cameron314/readerwriterqueue"
+    description = "A fast single-producer, single-consumer lock-free queue for C++"
+    topics = ("cpp11", "cpp14", "cpp17", "queue", "lock-free")
+    license = "BSD-2-Clause"
+    no_copy_source = True
+    settings = "os"
+    
+    @property
+    def _source_subfolder(self):
+        return "sources_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("readerwriterqueue-{}".format(self.version),
+                  self._source_subfolder)
+
+    def package(self):
+        self.copy("atomicops.h", src=self._source_subfolder, dst="include/moodycamel")
+        self.copy("readerwriterqueue.h", src=self._source_subfolder, dst="include/moodycamel")
```

Comment:
is the `moodycamel` part of the structure of the project?

Reply:
I was simply following a similar package (from the same author)'s path:
https://github.com/conan-io/conan-center-index/blob/master/recipes/concurrentqueue/all/conanfile.py#L25

If we correct it to `include/xxx.h`, I think we should change that one as well in a new version or something.

Reply:
From the README  https://github.com/cameron314/readerwriterqueue#cmake-installation

> Then, you can include it from your source code: `#include <readerwriterqueue/readerwriterqueue.h>`

You just need to fix the name

the other mine seems to be a mistake https://github.com/cameron314/concurrentqueue#basic-use